### ISSUE TITLE
CUDA error: compute capability 3.2 or greater required (available 0.1 for device

### ISSUE THREAD

**Issue by xialang2012** [cdcseacave/openMVS#96]
I downloaded the windows binary release from https://github.com/cdcseacave/openMVS_sample, but when I run the command
`DensifyPointCloud.exe -w tomvs -i tomvs/scene.mvs`
It showed

> 19:50:15 [App     ] Build date: Jan 18 2016, 12:21:46
> 19:50:15 [App     ] CPU: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz
> 19:50:15 [App     ] RAM: 31.94GB Physical Memory 128.00TB Virtual Memory
> 19:50:15 [App     ] OS: Windows 8.1 x64
> 19:50:15 [App     ] SSE & AVX compatible CPU & OS detected
> 19:50:15 [App     ] Command line: -w tomvs -i tomvs/scene.mvs
> 19:50:15 [App     ] MapSMtoCores for SM 1.2 is undefined; default to use 128 cores/SM
> 19:50:15 [App     ] CUDA error: compute capability 3.2 or greater required (available 0.1 for device
> [2])

I think this means I need a new card to run the code. I am wonder if there is a binary release version that I can used?

**Comment by cdcseacave** [comment]
Yes, I can disable the CUDA support and upload the new binaries. I will do it as soon as I find some free time.

**Comment by xialang2012** [comment]
Okay, thank you for you reply. When the binaries without the CUDA is ready, please tell me.

**Comment by cdcseacave** [comment]
please try the binaries on the master branch, they should be CUDA free

**Comment by xialang2012** [comment]
you mean I can obtain the binaries from the link https://github.com/cdcseacave/openMVS_sample

**Comment by xialang2012** [comment]
By the way, does the OpenMVS support the multi-tiff files?

**Comment by cdcseacave** [comment]
1 yes
2 what are ”multi-tiff”?

**Comment by xialang2012** [comment]
Than means a tiff file with bands more than 3.

**Comment by cdcseacave** [comment]
I see... and what should contain the extra bands (3+)?

**Comment by xialang2012** [comment]
In fact, what need is getting the orthophoto for each band, so, I have to texture each band in the tif file (I do not know is it necessary to texture every band).
Do you have some good ideas about get the orthophoto for each band?

**Comment by cdcseacave** [comment]
I see! You are right, texturing can be done only once and reuse the computed parameters to retexture with a different band. This is fairly easy, however not supported by OpenMVS yet. You are more than welcome to contribute with this functionality.

**Comment by xialang2012** [comment]
Thank you. In fact, I am a new one about the texture, so, cloud you give me some directions about the main parts which I should change.

**Comment by cdcseacave** [comment]
Please read this paper that describes quite close the algorithm used in OpenMVS: http://www.gcc.tu-darmstadt.de/media/gcc/papers/Waechter-2014-LTB.pdf

For what do you plan to use this functionality, agriculture?

**Comment by xialang2012** [comment]
Thank you. Yes, We have many hyperspectral images which will be used to monitor the agriculture application need to be stitching.

**Comment by cdcseacave** [comment]
I am already involved in a project doing exactly that. Interested in discussing this in more detail?

**Comment by xialang2012** [comment]
Of cause.

**Comment by xialang2012** [comment]
I have read this article in general, and I am not very clear to some detailed parts in this article.
Besides, I just wonder if you can show me more detail information about the project that you have already involved in.

**Comment by cdcseacave** [comment]
Sure, but we are getting out of topic here. Let's continue the discussion on email. Where can I reach you?

**Comment by xialang2012** [comment]
776528546@qq.com
